.github/workflows/e2e-tests-manual.yaml

```
@@ -30,14 +59,17 @@ jobs:
         - 'dps-symmetric-key'
         - 'dps-x509'
 
+      max-parallel: 10
+
     steps:
     - uses: 'actions/checkout@v1'
       with:
         ref: "${{ github.events.input.branch }}"
 
     - name: 'Run'
       run: |
-        ./ci/e2e-tests.sh ${{ matrix.test_name }}
+        timeout 15m ./ci/e2e-tests/test-run.sh '${{ matrix.test_name }}' &&
+        timeout 5m ./ci/e2e-tests/test-cleanup.sh '${{ matrix.test_name }}'
       env:
         BRANCH: "${{ github.events.input.branch }}"
```

```suggestion
        BRANCH: "${{ github.event.inputs.branch }}"
```

https://github.com/Azure/iot-identity-service/commit/b59977078be65a4cc5d981cc036ced13f69b5f18#diff-1659b614bde0bc41517180808c08167a818ef7fc68e0f12ee672dc5bc0326531